src/graph/ShowExecutor.cpp

```diff
@@ -218,18 +218,19 @@ void ShowExecutor::showTags() {
         auto value = std::move(resp).value();
         resp_ = std::make_unique<cpp2::ExecutionResponse>();
         std::vector<cpp2::RowValue> rows;
-        std::vector<std::string> header{"Name"};
+        std::vector<std::string> header{"ID", "Name"};
         resp_->set_column_names(std::move(header));
 
-        std::set<std::string> uniqueTags;
+        std::map<nebula::cpp2::TagID, std::string> tagItems;
```

Comment:
I think it’s better to put the name as the key in the map , it will order the name. Because users usually remember the name firstly, then through name to get the id. and the returns put the name in the first column.

Reply:
I think to order name is a good idea but no need to put the names in the first column. List number comes first is more appreciated to me.

Reply:
Currently ordered by `ID` is same order as created